imports/api/images.js

```diff
@@ -12,6 +12,8 @@ import Helpers from './helpers.js'
 
 import { _ } from 'underscore'
 
+import { Slingshot } from 'meteor/edgee:slingshot'
+
```

Comment:
Is the import necessary? Since this is the only change in the file, I'm guessing not?